public/video-ui/src/util/parseGridMetadata.js

```diff
@@ -0,0 +1,33 @@
+function parseMimeType(mimeType) {
+
+  //Normalise Mime Types coming from the grid.
```

Comment:
Perhaps the grid should be provided normalised mime types "So you don't have to"(tm)